features/step_definitions/registry.rb

```diff
@@ -385,8 +386,18 @@
 end
 
 Given /^default image registry route is stored in the#{OPT_SYM} clipboard$/ do |cb_name| 
+  ensure_admin_tagged
+  ensure_destructive_tagged
   org_proj_name = project(generate: false).name rescue nil
   cb_name ||= :registry_route
   cb[cb_name] = route('default-route', service('default-route',project('openshift-image-registry'))).dns(by: admin)
   project(org_proj_name) if org_proj_name
 end
+
+Given /^current generation number of#{OPT_QUOTED} deployment under #{QUOTED} is stored into#{OPT_SYM} clipboard$/ do |name,namespace,cb_name|
```

Comment:
@wzheng1, it looks like the name of the deployment is required?  If so then  `#{OPT_QUOTED}` should be `#{QUOTED}`.

Reply:
@wzheng1 perhaps we should add a method to `lib/openshift/deployment.rb` to support getting the `generation` field.

Reply:
@wzheng1 added helper with this PR https://github.com/openshift/verification-tests/pull/315

Reply:
@pruan-rht Thanks for the pr, it simplifies my definition!